apps/cli/src/commands/start.ts

```diff
@@ -237,7 +238,7 @@ export const createStartCommand = () => {
                 CARTESI_ROLLUPS_NODE_CPUS: cpus?.toString(),
                 CARTESI_ROLLUPS_NODE_MEMORY: memory?.toString(),
                 CARTESI_SDK_IMAGE: DEFAULT_SDK_IMAGE,
-                CARTESI_SDK_VERSION: options.runtimeVersion.toString(),
+                CARTESI_SDK_VERSION: runtimeVersion?.toString(),
```

Comment:
Do you still need the `?.toString()`. Value should be string, as there is a defined default value.